src/NUnitFramework/tests/TestContextTests.cs

```diff
@@ -367,7 +367,7 @@ public void FilePathAndDescriptionDoesNotThrow()
 
         [TestCase(null)]
 #if !NETSTANDARD1_3 && !NETSTANDARD1_6
-        [TestCase("bad<>path.png", IncludePlatform = "Windows")]
+        [TestCase("bad<>path.png", IncludePlatform = "Win")]
```

Comment:
I filed https://github.com/nunit/nunit/issues/2201 about the fact this wasn't an error, but yes, PlatformAttribute definitely needs some work! An enum would be great.